style.css

```
@@ -40,3 +40,88 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+h2 {
+  padding-left: 15px;
+  font-size: 1.2rem;
+}
+
+.wait-list{
+  height: 250px;
+  overflow:auto; 
+  
+}
+
+.complete-list{
+  height: 250px;
+  overflow:auto;
+  color: grey;
+  text-decoration: line-through;
+}
+
+img{
+  float: right;
+  margin-right: 20px;
+}
+
+li{
+  list-style:none;
+  padding-left: 15px;
+  line-height: 30px;
+  width: 350px;
+}
+
+
+.input-box {
+  
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.25);
+  padding-bottom:0px;
+}
+
+.add-button{
+  width: 34px;
+  height: 34px;
+  border-radius: 34px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  background-color: #2ec1ac;
+  color: white;
+  -webkit-text-fill-color: white;
+  font-weight: 100;
+  margin: 15px;
+}
+
+#write-here{
+ 
+  padding: 10px;
+  height: 20px;
+  margin-left: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  text-align: center;
+
+  border: solid 1px rgba(0, 0, 0, 0.25);
+  border-radius: 20px;
+  width: 350px;
+  
+  display: flex;
+  flex-direction: row;
+  align-items: stretch; 
```

관련성 있는 CSS 끼리 묶어주면 더 좋을 것 같아요
```suggestion
  
  width: 350px;
  height: 20px;
  
  padding: 10px;
  margin-left: 10px;
  margin-top: 10px;
  margin-bottom: 10px;
  
  text-align: center;

  border: solid 1px rgba(0, 0, 0, 0.25);
  border-radius: 20px;
  
  display: flex;
  flex-direction: row;
  align-items: stretch; 
```